### ISSUE TITLE
A11y_Azure Data Studio_Toolbar_FindNode_Label: Visual label is not provided for combo boxes in the Find node pop up.

### ISSUE THREAD

**Issue by Truptik12** [microsoft/azuredatastudio#20781]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #HeadingsandLabels; #WCAG2.4.6; #FTP;

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:

1. Launch the "Azure data studio" application.
2. Connect with the provided "sqltools2019-3" SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. TAB to the toolbar present at right side of the screen.
7. TAB to "Find Node" button and hit ENTER key.

### Actual:
Visual label is not provided for combo boxes in the Find node pop up.

### Expected Result:
Visual label should provided for combo boxes in the Find node pop up.

### Attachment:
![image](https://user-images.githubusercontent.com/86710491/194818046-9e583727-7af6-4a20-8b50-8fa888d1a494.png)

**Comment by aasimkhan30** [comment]
@Truptik12, this issue is fixed in the latest insiders. Can you please re test and close this issue?

**Comment by Truptik12** [comment]
@aasimkhan30 

Verified the bug in the below environment
Version: 1.40.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/d5cf088ba7b794859438fd2c1902249fd1e6a947
Date: 2022-10-26T05:45:47.459Z
VS Code: 1.67.0
Electron: 17.4.5
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Windows_NT x64 10.0.22621

The bug is not yet fixed, the combo box still doesn't have visual labels

**Comment by aasimkhan30** [comment]
@Truptik12 
we are using similar patterns across ads which is inherited from Vscode. We are providing users with the aria lables for those controls. Is that not enough?

Search example:
![image](https://user-images.githubusercontent.com/6816294/199164665-e96e1617-7ce5-4e7c-871a-feaeb238203a.png)

**Comment by aasimkhan30** [comment]
@Truptik12 , based on the guidelines WCAG2.4.6 https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html
It seems that visual labels are not necessary.
![image](https://user-images.githubusercontent.com/6816294/199414605-b52521be-a717-42e9-8c6c-db8fde8dac69.png)

If that's the case then this is not a bug. Can you please verify and close this issue?

**Comment by Truptik12** [comment]
Closing the bug as we have logged the MAS under the wrong WCAG standard.
Please find the bug with the correct WCAG standard: https://github.com/microsoft/azuredatastudio/issues/21414.

**Comment by Rama8345** [comment]
Github Tag: #Closed
